src/main/python/vyu/tracker.py

```diff
@@ -1,14 +1,32 @@
 from contextlib import contextmanager
+import imageio
+import numpy as np
+
+from vyu.image import image2position
+from vyu.timer import Timer
 
 
 class EyeTracker(object):
 
-    def __init__(self):
+    def __init__(self, camera='<video0>'):
         self.transform_matrix = np.eye(2)
         self.transform_bias = np.zeros(2)
```

Comment:
What's transform bias

Reply:
Should be addressed with commit
7da4f88a0ee856be715ded2cb5078901ed5149bb